docs/index.rst

```diff
@@ -0,0 +1,7 @@
+=====================================
+numFOCUS Infrastructure Documentation
+=====================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
```

Comment:
This probably requires explicit links to the project requests for things to show up here?

Reply:
Yeah, I will do that now.